Lib/test/ann_module.py

```diff
@@ -51,3 +52,8 @@ def foo(x: int = 10):
     def bar(y: List[str]):
         x: str = 'yes'
     bar()
+
+def dec(func):
+    def wrapper(*args, **kwargs):
```

Comment:
Any reason why this doesn't use `wraps` as a decorator?

```suggestion
    @wraps(func)
    def wrapper(*args, **kwargs):
```